modules/web/src/app/shared/entity/common.ts

```diff
@@ -41,22 +41,27 @@ export type ResourceLabelMap = {
 export enum View {
   Overview = 'overview',
   Clusters = 'clusters',
+  ExternalClusters = 'external',
   ClusterTemplates = 'clustertemplates',
   Projects = 'projects',
   Members = 'members',
+  Groups = 'groups',
   SSHKeys = 'sshkeys',
   ServiceAccounts = 'serviceaccounts',
   Wizard = 'wizard',
   Account = 'account',
   Settings = 'settings',
   MachineDeployment = 'md',
   Backups = 'backups',
+  Snapshots = 'snapshots',
+  Restores = 'restores',
 }
 
 // IMPORTANT: names in 'ViewDisplayName' have to be in sync with names in 'View'
 export enum ViewDisplayName {
   Overview = 'Overview',
   Clusters = 'Clusters',
+  ExternalClusters = 'External Clusters',
```

Comment:
Add missing

  Snapshots = <DISPLAY_NAME>
  Restores = ,<DISPLAY_NAME>

and utilize it just alike others are used IF NEEDED